website/members/services.py

```diff
@@ -229,12 +229,12 @@ def execute_data_minimisation(dry_run=False, members=None) -> list[Member]:
             profile = member.profile
             profile.student_number = None
             profile.phone_number = None
-            profile.address_street = "<removed> 1"
+            profile.address_street = None
             profile.address_street2 = None
-            profile.address_postal_code = "<removed>"
-            profile.address_city = "<removed>"
-            profile.address_country = "NL"
-            profile.birthday = datetime(1900, 1, 1)
+            profile.address_postal_code = None
+            profile.address_city = None
+            profile.address_country = None
+            profile.birthday = None
```

Comment:
In website/members/models/profile.py line 276-277 the `is_minimised` property should also be rewritten now

Reply:
> In website/members/models/profile.py line 276-277 the `is_minimised` property should also be rewritten now

Or maybe we should introduce a `is_minimised` field, and clean the model to only allow null fields if `is_minised` is `True`